### PR TITLE
Change index page to landing page

### PR DESCRIPTION
**Pull request by ZachGlassman**
Addresses https://github.com/thedataincubator/forecaster/issues/6

### CODE REVIEW

**Comment by dzbee** [comment]
Hi Zach!

**Comment by craw-daddy** [comment]
Ok, so my understanding here is that the html filename would have to be changed from "index.html" to "predict.html" for this change to actually be effective.  (My understanding of Flask is growing rather slowly.)

Or is it that the function name for the route needs to be changed?  So

@app.route('/predict')
def predict():
   ...

**Comment by ZachGlassman** [comment]
The template rendered is controlled by the `render_template` function, the route itself does not necessarily need to be related

**Comment by craw-daddy** [comment]
Sorry, maybe I edited my comment after you saw it...  (Bad practice...)

Does the name of the function that defines the route need to match the route?  

@app.route('/predict')
def predict():
...

**Comment by dzbee** [comment]
Nope! `@app.route('/whatever')` simply says that the URL/endpoint will be [domain]/whatever (e.g. www.example.com/whatever). The function name and the template name used in `render_template()` are free be whatever you like, though it is conventional that these three things match.

**Comment by craw-daddy** [comment]
Ok, then I guess I don't understand what is causing the tests to fail in this case.  Any hint?

**Comment by ZachGlassman** [comment]
The hint is that my commit on this branch caused the tests to fail.  The tests check that there is something at `/`.

**Comment by craw-daddy** [comment]
Ok, well given that, I'd tend to reject this change since the tests are failing.  (I thought that the idea of accepting a change is that the result will be "deployable" after the change is accepted.)

**Comment by ZachGlassman** [comment]
Valid point in general, however I would like you to write code to make the tests pass.  Sort of like test driven development.

**Comment by craw-daddy** [comment]
Ok, let's approve this and I'll work on issue #6 to create an index page.

**Comment by ZachGlassman** [comment]
please do so on this branch and when tests pass we will review.

**Comment by ZachGlassman** [comment]
this looks good, tests are failing because they are testing the `/` route for predictions but its no longer the predict route, so you will need to change them to hit the /predict route.

**Comment by craw-daddy** [comment]
Ok, I am pretty certain that I'm not understanding everything that's going on here with the tests.  ;)

**Comment by ZachGlassman** [comment]
The tests are mostly passing, you will need to fix a few things for pylint and then this is good!

forecaster/forecaster.py:15:0: E0011: Unrecognized file option 'diasable' (unrecognized-inline-option)
forecaster/forecaster.py:15:4: W0612: Unused variable 'landing' (unused-variable)

so I think the first one is just a typo (diasable != disable) and the second one might also disappear if that first one is fixed.

File: forecaster/forecaster.py [hunk 1]
```diff
@@ -12,6 +12,11 @@ def create_app(prophet_url, secret_key, bokeh_version):
     app = Flask(__name__)
 
     @app.route('/')
+    def landing():  # pylint: diasable=W0612
```
Comment (ZachGlassman) [inline forecaster/forecaster.py]: spelling

**Comment by craw-daddy** [comment]
Checks have passed now.  The merge conflict needs to be resolved.  But I think this resolves this issue, as well as Issue 6.

**Comment by dzbee** [comment]
I'll let @ZachGlassman voice his own opinion here, but usually the way I address conflicts is to locally rebase the branch on master (resolving the conflicts locally during the rebase), close the current PR and delete the remote branch, and then push the local branch to the remote and open a new PR.

**Comment by ZachGlassman** [comment]
Please do as Dylan suggested and assign me to merge.

**Pull request closed without merging**